pkg/component/observability/monitoring/alertmanager/alertmanager.go

```diff
@@ -73,6 +73,11 @@ func (a *alertManager) alertManager(takeOverOldPV bool) *monitoringv1.Alertmanag
 		obj.Spec.AlertmanagerConfiguration = &monitoringv1.AlertmanagerConfiguration{Name: a.name()}
 	}
 
+	if a.values.Replicas > 1 {
+		obj.Spec.PodMetadata.Labels["networking.resources.gardener.cloud/to-alertmanager-operated-tcp-9094"] = v1beta1constants.LabelNetworkPolicyAllowed
+		obj.Spec.PodMetadata.Labels["networking.resources.gardener.cloud/to-alertmanager-operated-udp-9094"] = v1beta1constants.LabelNetworkPolicyAllowed
```

Comment:
Why is it `to-alertmanager-operated-...`? I would have expected `to-alertmanager-garden-...` when looking at the garden operator registration code and the corresponding name implied in the `service`.
`to-alertmanager-operator-...` would probably also be fine from my point of view.

Reply:
TBH, I have no idea. `prometheus-operator` is generating this service for the meshes. I don't see that it can be influenced via configuration.

Reply:
Here is some information about this "governing service":
- https://github.com/prometheus-operator/prometheus-operator/blob/f8c181b974cb83c92308b732ed24e9b75c4f9171/Documentation/rbac.md?plain=1#L142
- https://github.com/prometheus-operator/prometheus-operator/issues/4812#issuecomment-1137221108
- https://github.com/prometheus-operator/prometheus-operator/issues/3805